hasher-matcher-actioner/hmalib/storage/dynamostore.py

```diff
@@ -1,19 +1,22 @@
 from mypy_boto3_dynamodb.service_resource import Table
 
-class _IDynamoDBItem():
+
+class _IDynamoDBItem:
```

Comment:
Nit: It's generally understood in python that prefixing module or class-level variable with an underscore means that it should be treated as "private", internal-details, or may be refactored in the future without warning.